### DISCUSSION TITLE
How to deploy the static version of Nuxt 3? npm run generate still serves on Universal mode

### DISCUSSION THREAD

**Discussion by itszooper** [nuxt/nuxt#16943] [Questions]
I just deployed a nuxt 3 site on netlify using the command `npm run generate` but I can still see that the SSR version of the site is being served from netlify. Can someone help me on this? I even checked it with vercel, still the same.

![Screenshot2022-08-03 at 17 23 44](https://user-images.githubusercontent.com/107394587/182601554-521bc38d-c4da-4ccc-8628-63b3622b9269.png)

![image](https://user-images.githubusercontent.com/107394587/182601494-95269df4-38a3-4765-a55a-13e7e45120c0.png)

In the [docs](https://v3.nuxtjs.org/guide/deploy/static-hosting#prerendering) it is mentioned the output directory is `.output/public`, I even set it to this in Netlify, but I get this error
![Screenshot2022-08-03 at 17 29 27](https://user-images.githubusercontent.com/107394587/182602486-904d36b0-5202-4243-a374-3acc06bbd376.png)

I even tried to do `npm run generate` locally and serve the html file generated in `.output/public` it just starts opens broken webpage, with just text and images css and js do not work.

Also, why are there two folders getting generated `dist` and `.output/public` when we run the generate build
![Screenshot2022-08-03 at 17 46 15@2x](https://user-images.githubusercontent.com/107394587/182605665-512dc814-8ce7-43b0-9421-ba2059873d43.png)

@danielroe @pi0  please help.

**Comment by dylankelly** [comment]
@itszooper Im having the exact same issue. I suspect this may have something to do with the nitro preset https://nitro.unjs.io/deploy/providers/netlify but I have no idea how that works.
